src/commands/clear.rs

```diff
@@ -3,17 +3,18 @@ use bevy::prelude::*;
 use crate as bevy_console;
 use crate::console::ConsoleState;
 use crate::ConsoleCommand;
+use clap::Parser;
 
 /// Clears the console
-#[derive(ConsoleCommand)]
-#[console_command(name = "clear")]
+#[derive(Parser, ConsoleCommand)]
+#[command(name = "clear")]
 pub(crate) struct ClearCommand;
 
 pub(crate) fn clear_command(
     mut clear: ConsoleCommand<ClearCommand>,
     mut state: ResMut<ConsoleState>,
 ) {
-    if clear.take().is_some() {
+    if let Some(Ok(_)) = clear.take() {
```

Comment:
```suggestion
    if clear.take().is_okay() {
```

Reply:
this doesn't seem to work, do watch out this is a `Option<Result<_>>` nowadays

Reply:
Ah right. You could do .is_some().is_okay(), but I'm not convinced that's clearer.

Reply:
Hmm, yeah guess I'll leave it as it is for now!